initramfs-tools/keyscript.sh

```diff
@@ -11,4 +22,9 @@ if [ "$FIDO2LUKS_USE_TOKEN" -eq 1 ]; then
 	export FIDO2LUKS_CREDENTIAL_ID="$FIDO2LUKS_CREDENTIAL_ID,$(fido2luks token list --csv $CRYPTTAB_SOURCE)"
 fi
 
-fido2luks print-secret --bin
+fido2luks print-secret --bin 2>/dev/null
```

Comment:
Hiding potential error messages isn't really helpful please remove `2>/dev/null

Reply:
True, remnant from testing when I was exploring different approaches, including saving stdout/stderr to variables. Saving binary output of fido2luks to shell variables turned out unfeasible and I overlooked that I had left the '2>/dev/null' in the script. Thanks for spotting and merging.